src/App.js

```diff
@@ -124,8 +126,11 @@ function App() {
           <Route path="/login" element={<Login />} />
           <Route path="/ForgotPassword" element={<ForgotPassword />} />
           <Route path="/signup" element={<Signup2 />} />
+          <Route path="/userlist" element={<UserList/>}/>
           <Route path="/allnews" element={<Newscard />} />
           <Route path="/allnews/:id" element={<Readmorenews />} />
+          <Route path="/" element={<UserList />} />
```

Comment:
Why is there a '/' path here for user list? you already have the user list under the userlist path

Reply:
I was testing something else- now it 's fixed.